<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetForm</class>
 <widget class="QWidget" name="SetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>350</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="SelectPlugin" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>15</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="SelectPlugin" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="SelectPlugin" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>65</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="SelectPlugin" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="SelectPlugin" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>115</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="SelectPlugin" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="SelectPlugin" name="widget_7" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>165</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="SelectPlugin" name="widget_8" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>215</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SelectPlugin</class>
   <extends>QWidget</extends>
   <header location="global">selectplugin.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
